### PR TITLE
get_new_version added and atomatic push parameter

### PR DESCRIPTION
**Pull request by azime**
- Add new end point "githubflow_get_new_vesion" : Calculate new version
- Add new parameter "--auto-push": Automatic push

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,9 +34,15 @@ In the dedicated python 3 virtualenv, inside repository:
 
 ```bash
 pip install -r requirements.txt          # install dependencies only
-python githubflow_release/run.py --help  # invoke
+python githubflow_release --help # invoke
```
Comment (pbougue) [inline README.md]: Not sure: This change looks weird to me.

Did you notice that there are 2 ways to invoke the same script that are described here? `install + <cmd>` (above) or `python <file>` (this one).
If you tried both from root of repo, then OK.

Above, an update of the `install + <cmd>` might be required too, as you introduce a new cmd.

File: README.md [hunk 2]
```diff
@@ -34,9 +34,15 @@ In the dedicated python 3 virtualenv, inside repository:
...
 ```
 
+Or
+
+```bash
+pip install -r requirements.txt          # install dependencies only
+python githubflow_get_new_version --help # invoke
```
Comment (pbougue) [inline README.md]: Not sure, I would have expected this here:
```suggestion
python githubflow_release/new_version.py --help # invoke
```

File: README.md [hunk 3]
```diff
@@ -56,11 +62,27 @@ generate_debian_changelog: False
...
+cd my_project;
+githubflow_release --release-type major|minor --github-repo User/repo_name --project-path /path/repo_name/ --remote-name origin --auto-push
+```
+
+Get new version (in the dedicated virtualenv):
 
 ```bash
 cd my_project;
-githubflow_release --release-type minor
+githubflow_get_new_version --release-type major|minor|hotfix --project-path /path/repo_name/ --remote-name origin
```
Comment (pbougue) [inline README.md:85]: minor: Can you add an example of the output? (maybe the display the current version before via git describe or something?)

**Review by pbougue — COMMENTED** [review]
Except for doc on invocations that I didn't get (and I would like to keep an easy first-use as this is critical to release), the rest is good to me :+1:

File: README.md (hunk 1 above)
Comment (azime) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (azime) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (azime) [reply README.md:85]: Done :+1:

**Review by xlqian — APPROVED** [review]

**Review by datanel — APPROVED** [review]

**Pull request merged**